modules/vocabularies/src/main/frontend/src/vocabulariesAdminPage.jsx

```diff
@@ -18,17 +18,20 @@
 //
 
 import {
+  Button,
+  Dialog,
+  DialogContent,
   Grid,
-  Typography
+  TextField,
+  Typography,
+  makeStyles
 } from "@material-ui/core";
 
-import React from "react";
+import React, {useEffect} from "react";
 
 import VocabularyDirectory from "./vocabularyDirectory";
-
 import OwlInstaller from "./owlInstaller";
-
-import fetchBioPortalApiKey from "./bioportalApiKey";
+import { fetchBioPortalApiKey, BioPortalApiKey } from "./bioportalApiKey";
```

Comment:
Fetch is no longer used directly:

```suggestion
import { BioPortalApiKey } from "./bioportalApiKey";
```